sources/findAllSequenceMultiProcess.c

```diff
@@ -0,0 +1,70 @@
+#include <sys/sysinfo.h>
+#include <signal.h>
+#include <wait.h>
+#include "findAllSequence.h"
+
+unsigned int* findAllSequence(const char *filepath,
+                              char **sequenceArray, size_t size) {
+    errno = 0;
+    fileMapping *mapping = readFile(filepath);
+    if (mapping == NULL) {
+        // check errno
+        return NULL;
+    }
+    size_t dataSize = strlen(mapping->dataPtr);
+    dataInfo dInfo = {mapping->dataPtr, dataSize};
+    int procCounter = get_nprocs();
+
+    unsigned int* ptrSeqCounter = mmap(NULL, size*sizeof(unsigned int),
+                               PROT_READ | PROT_WRITE,
+                               MAP_SHARED | MAP_ANONYMOUS,
+                               0, 0);
```

Comment:
-1, 0

some implementations require fd to be -1 if MAP_ANONYMOUS (or MAP_ANON) is specified, and portable applications should ensure this